### PR TITLE
Add rotation examples. Fix #25

### LINKED ISSUE #25: Rotation example page
Now that @robhobbes's https://github.com/openseadragon/openseadragon/pull/185 has landed, we should add an example page for the rotation feature.

### PR DESCRIPTION
**Pull request by avandecreme**

### CODE REVIEW

File: www/ui-rotation.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        pinchRotate: true
+    }
+});
+    </pre>
+</div>
+
+<div class="description">
+    <h3>Programmatically set the rotation angle</h3>
+    <p>
+        To have more control hover the rotation, you can set it via
```
Comment (iangilman) [inline www/ui-rotation.html]: "over" not "hover"

File: www/ui-rotation.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        </noscript>
+    </div>
+    <pre>
+var viewer = OpenSeadragon({
+    id:            "example-programmatically",
+    prefixUrl:     "/openseadragon/images/",
+    tileSources:   "/example-images/highsmith/highsmith.dzi"
+});
+
+$("#slider").slider({
```
Comment (iangilman) [inline www/ui-rotation.html]: Add a comment here to say you're using jQuery UI for the slider.

**Comment by iangilman** [comment]
You're on a roll!

**Comment by avandecreme** [comment]
>You're on a roll! 

Yup! :D

**Comment by iangilman** [comment]
Deployed: 

http://openseadragon.github.io/examples/ui-rotation/

I love the slider demo...pretty slick :)

**Pull request merged**